src/components/OSCALControl.js

```diff
@@ -41,6 +41,32 @@ const useStyles = makeStyles(() => ({
   OSCALControlStatus: (props) => getControlStatusCss(props),
 }));
 
+/**
+ * Retrieve the set-parameters when contained in the top-level implemented requirement statement
+ *
+ * @param {Object} implReqStatements
+ * @param {String} componentId
+ * @returns An array of set-parameters
+ */
+function getImplReqSetParameters(implReqStatements, componentId) {
```

Comment:
Should this entire function be pushed down to `OSCALControlProse.js` since it's already responsible for finding the parameters and has the `componentId`?

Reply:
I will push the function to `OSCALControlProse.js`